content/en/metrics/nested_queries.md

```diff
@@ -0,0 +1,205 @@
+ 
+
+{{% collapse-content title="Percentiles in Multilayer Space Aggregation example query" level="h5" %}}
+
+We can use percentiles in multilayer space aggregation to additionally summarize the results of our nested query (avg CPU utilization by `env` and `team` every 5 minutes) by calculating the p95th value of this nested query for every unique `env` value. 
+
+In the UI or JSON tab, it would look as follows:
+
+ {{< img src="/metrics/nested_queries/percentiles-space-agg-ui.png" alt="example of MLA space agg percentiles using nested queries in the UI" style="width:100%;" >}}
+
+ {{< img src="/metrics/nested_queries/percentiles-space-agg-json.png" alt="example of MLA space agg percentiles using nested queries in the JSON" style="width:100%;" >}}
+
+  {{% /collapse-content %}} 
+
+
+
+{{% collapse-content title="Standard deviation example query" level="h5" %}}
+
+Standard deviation helps measure the variability or dispersion of a dataset. The following query uses standard deviation with multilayer time aggregation to calculate the standard deviation of our nested query (sum of API request counts, averaged over 4 hour) over longer twelve-hour periods:
+
+In the UI or JSON tab, it would look as follows:
+
+ {{< img src="/metrics/nested_queries/nested-queries-std-ui.png" alt="example of standard deviation with nested queries in the UI" style="width:100%;" >}}
+
+ {{< img src="/metrics/nested_queries/nested-queries-std-json.png" alt="example of standard deviation with nested queries in the JSON" style="width:100%;" >}}
+{{% /collapse-content %}} 
+
+
+## Higher Resolution Queries
+
+As a reminder, every metric query contains an initial layer of time aggregation (aka. rollup) which controls the granularity of datapoints plotted on a graph. Datadog provides default rollup time intervals that grow as your overall query timeframe grows. With nested queries, you can now access more granular, high-resolution data over longer, historical timeframes.
+
+ {{< img src="/metrics/nested_queries/higher-res-query-example.png" alt="example of standard deviation with nested queries in the UI" style="width:100%;" >}}
+
+{{% collapse-content title="Higher resolution example query" level="h5" %}}
+
+Historically when querying a metric over the past month, you would see data at 4-hour granularity. You can now use nested queries to access higher granularity data over this historical timeframe. Here's an example that calculates the standard deviation of high-resolution metrics batch counts, with a 5-minute base and a 4-hour reducer interval.
+
+_Note: We recommend you define your initial rollup with the most granular rollup interval and use multilayer time aggregation with coarser rollup intervals to get more user-readable graphs._
+
+In the UI or JSON tab, it would look as follows:
+
+{{< img src="/metrics/nested_queries/nested-queries-higher-res-ui.png" alt="example of higher resolution queries using nested queries in the UI" style="width:100%;" >}}
```

Comment:
This image needs to be replaced to match the description above -- higher resolution queries are the most valuable over historical timeframes (>2d) (not the realtime timeframe shown in this screenshot)

Reply:
Gotcha, thanks for the callout here. I've updated the example to query for the past month instead of the past day.